tests/task_comments/test_api_create_task_comment.py

```diff
@@ -76,6 +77,20 @@ def test_task_comment_creation_api_with_relation_not_existing(self):
         self.assertEqual(HTTPStatus.NOT_FOUND.value, actual_response.status_code)
         self.assertDictEqual(expected_response, json.loads(actual_response.data))
 
+    def test_task_comment_creation_api_with_unaccepted_relation(self):
+        auth_header = get_test_request_header(self.admin_user.id)
+        expected_response = messages.UNACCEPTED_STATE_RELATION
+        actual_response = self.client.post(
+            f"mentorship_relation/{self.relation_id_one}/task/{self.task_id}/comment",
```

Comment:
Here self.task_id is 1 and task 1 is is in `mentorship_relation_w_second_user`. You need to create another task that is in `task_list_3` 
Also, the `auth_header` should use second user id instead of admin user id

Reply:
Creating a new task in ```task_list_3``` is breaking other existing tests. Therefore I have created a new mentorship relation for this test

Reply:
Would be great if you add screenshots of what are the test cases that breaks.

Reply:
![Capture](https://user-images.githubusercontent.com/35465871/108620780-6aa1de00-7454-11eb-9ee6-4c4d096a83b2.PNG)

Reply:
Thanks for clearing :raised_hands: